dataStructures/queues/queue.go

```diff
@@ -0,0 +1,74 @@
+package main
```

Comment:
It's good practice to provide package-level comments even if it's a binary. This is because www.godoc.org will use them. [Some more info on the standard format](https://github.com/golang/go/wiki/CodeReviewComments#package-comments).

Reply:
@aschlosberg Thank you for the feedback, I am currently studying effective go to intimate myself.